gstreamer-video/src/video_rectangle.rs

```diff
@@ -0,0 +1,35 @@
+// Copyright (C) 2017 Philippe Normand <philn@igalia.com~
+//
+// Licensed under the Apache License, Version 2.0 <LICENSE-APACHE or
+// http://www.apache.org/licenses/LICENSE-2.0> or the MIT license
+// <LICENSE-MIT or http://opensource.org/licenses/MIT>, at your
+// option. This file may not be copied, modified, or distributed
+// except according to those terms.
+
+use ffi;
+
+#[derive(Debug)]
```

Comment:
This probably should get a few more traits derived. PartialEq, Eq for example. And you might want to implement the GLib conversion traits

Reply:
Actually the conversion traits wouldn't make much sense, considering how it is used. That would only work if you made it a wrapper around the FFI type.

Reply:
Ok, PR updated :)